python/src/iceberg/expression/literals.py

```diff
@@ -51,7 +52,12 @@
     UUIDType,
 )
 
-EPOCH = datetime.datetime.utcfromtimestamp(0)
+
+EPOCH_DATE = date.fromisoformat("1970-01-01")
+EPOCH_TIMESTAMP = datetime.fromisoformat("1970-01-01T00:00:00.000000")
+ISO_TIMESTAMP = re.compile(r"\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(.\d{1,6})?")
+EPOCH_TIMESTAMPTZ = datetime.fromisoformat("1970-01-01T00:00:00.000000+00:00")
```

Comment:
nit, maybe worth renaming it to `EPOCH_TIMESTAMP_TZ` and `ISO_TIMESTAMP_TZ`

Reply:
I think it's fine since these are usually one word.